### PR TITLE
Introduce slug argument in CLI

### LINKED ISSUE #644: New parameter in CLI to specify the correct slug
As @dd32 said in [#441 comment](https://github.com/WordPress/plugin-check/issues/441#issuecomment-2357911990), we need to specify the correct slug in the wp cli command, so we could compare with the ones that we have in the plugin.

It's related with PR #441

### PR DESCRIPTION
**Pull request by ernilambar**
Fixes https://github.com/WordPress/plugin-check/issues/644

### CODE REVIEW

**Comment by davidperezgar** [comment]
@ernilambar did you test it the i18n?

**Comment by ernilambar** [comment]
> @ernilambar[](https://profiles.wordpress.org/rabmalin) did you test it the i18n?

No. We can implement it in other checks later after proper slug is available.

File: includes/Checker/Abstract_Check_Runner.php [hunk 1]
```diff
@@ -156,6 +165,15 @@ abstract protected function get_include_experimental_param();
 	 */
 	abstract protected function get_categories_param();
 
+	/**
+	 * Returns plugin slug parameter.
+	 *
+	 * @since 1.0.0
```
Comment (mukeshpanchal27) [inline includes/Checker/Abstract_Check_Runner.php]:
```suggestion
	 * @since 1.2.0
```

**Review by mukeshpanchal27 — DISMISSED** [review]
The PR introduce the warning `textdomain_mismatch` for AJAX runner. It return wrong slug for plugin.

Screenshot 2024-09-20 at 12 31 36 PM

Make sure to add unit tests this scenario if possible

**Comment by ernilambar** [comment]
PR has been updated.

**Review by swissspidy — APPROVED** [review]

**Review by davidperezgar — APPROVED** [review]

**Pull request merged**